src/test/java/core/basesyntax/handler/impl/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.handler.impl.impl.AddOperationHandler;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private static OperationHandler addOperationHandler;
+    private static Fruit fruit;
+    private static Transaction transaction;
+
+    @BeforeClass
+    public static void beforeClass() {
+        addOperationHandler = new AddOperationHandler();
+        fruit = new Fruit("apple");
+        transaction = new Transaction("s", "apple", 15);
+    }
+
+    @After
+    public void tearDown() {
+        Storage.fruitStorage.clear();
+    }
+
+    @Test
+    public void addHandlerUsualState_ok() {
```

Comment:
```suggestion
    public void addHandler_UsualState_ok() {
```better use this pattern. It's more clear. Check  all test names